setup/utils/build_protos.py

```diff
@@ -21,16 +21,27 @@
 
 
 def is_grpc_service_dir(files):
-    """Returns true iff the directory hosts a gRPC service."""
+    """Returns true iff the directory hosts a gRPC service.
```

Comment:
If we are taking the opportunity to tidy up things, then this file would be better named as ```protos_builder.py```.

Then, the following code usage would be so much cleaner.  i.e. no import as, and the caller would be much more readable, as protos_builder.clean().
```
from setup.utils import build_protos as forseti_protos

forseti_protos.clean(abs_path)
forseti_protos.make_proto(abs_path)
```

Reply:
done!